net-libs/usrsctp/usrsctp-0.9.5.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="A cross-platform userland SCTP stack"
+HOMEPAGE="https://github.com/sctplab/usrsctp"
+SRC_URI="https://github.com/sctplab/usrsctp/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DOCS=( LICENSE.md Manual.md README.md )
+PATCHES=( "${FILESDIR}/pc-inc-path-${PV}.patch" )
```

Comment:
Please include the package name in filename also, 
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#file-naming

Rest looks good.

Reply:
thanks. fixed